### PR TITLE
Aswathy/DPROD-3488/Deriv Go product page redirection issue

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   Deriv Go product page redirection issue

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/deriv-go/_banner.tsx [hunk 1]
```diff
@@ -112,6 +110,42 @@ const DHero = () => {
     const [is_logged_in] = useAuthCheck()
     const is_rtl = useIsRtl()
 
+    const getMobileOperatingSystem = () => {
```
Comment (habib-deriv) [inline src/pages/deriv-go/_banner.tsx:113]: Lets move this function in utility

**Review by hasan-deriv — APPROVED** [review]

File: src/pages/deriv-go/_banner.tsx (hunk 1 above)
Comment (fasihali-deriv) [inline src/pages/deriv-go/_banner.tsx:113]: Hi @aswathy-deriv I guess we have this function. 
Can you please check this file if its useful for you. `/src/common/os-detect.ts`

Comment (aswathy-deriv) [reply src/pages/deriv-go/_banner.tsx:113]: yeah its already there.okay i will use it from there then.thanks @fasihali-deriv

**Review by habib-deriv — DISMISSED** [review]

**Comment by aswathy-deriv** [comment]
Closing this PR.Please refer to this [https://github.com/binary-com/deriv-com/pull/7386](url)

**Pull request closed without merging**